corehq/apps/export/models/new.py

```diff
@@ -1120,10 +1123,19 @@ def identifier(self):
 
     @classmethod
     def _new_from_schema(cls, schema):
-        return cls(
-            domain=schema.domain,
-            case_type=schema.case_type,
-        )
+        settings = get_or_create_default_export_settings_for_domain(schema.domain)
```

Comment:
Look like you and @biyeun already discussed this naming, but my take is that seeing `get_or_create` makes me surprised to then see `None` be returned, since the similarly-named django ORM method always return something. `get_or_create_default_export_settings_for_domain_unless_not_allowed` would be a bit much. What about `get_default_export_settings_if_available` or even just `get_default_export_settings`?

Reply:
I went with `get_default_export_settings_for_domain`. Thought it might be nice to be more explicit and start with `try_to_get...` or something similar but I think this is sufficient. Thanks for the feedback.